Wellthier/DocManagers/ExerciseDoc.h

```diff
@@ -0,0 +1,23 @@
+//
+//  ExerciseDoc.h
+//  Wellthier
+//
+//  Created by Craig Lee on 8/1/22.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface ExerciseDoc : NSObject
```

Comment:
We need to think better naming convention

Reply:
Something like ExerciseDatabase?